### ISSUE TITLE
Support jQuery event handler registeration style.

### ISSUE THREAD

**Issue by Zodiase** [Zodiase/meteor-infinite-load#10]
Such as:

``` JavaScript
// Client Side.
var infiniCollection = InfiniLoad(collection);
infiniCollection.on('ready', someHandler1); // Trigger someHandler1 when "ready".
infiniCollection.on('ready', someHandler2); // Can bind multiple handlers to some event.
infiniCollection.on('update', someHandler3); // Trigger someHandler3 when "update".
infiniCollection.off('ready', someHandler1); // Unbind a handler for some event.
infiniCollection.off('update'); // Unbind all handlers for some event.
infiniCollection.one('ready', someHandler4); // Bind a handler for some event that is triggered at most once.
```

**Comment by Zodiase** [comment]
Probably go with `.onReady(func)` and `.onUpdate(func)` first.

**Comment by Zodiase** [comment]
Resolved with 1d08979aab573bc9f292886830293da7ab461b7d
